api/appsettings.json

```diff
@@ -15,5 +15,8 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "HotelsDb": "Server=(localdb)\\mssqllocaldb;Database=Hotels.Database;Trusted_Connection=True;MultipleActiveResultSets=true"
+  },
+  "Resources": {
+    "ImagesFolder": "C:\\Users\\Tatsiana.Maltsava\\Desktop\\Hotel\\api\\Resources\\Images"
```

Comment:
Is Desktop folder a good place to store your files? ;)

Reply:
Which folder would you prefer? I could easily change files location folder now))